fem/estimators.hpp

```diff
@@ -426,13 +425,13 @@ class LpErrorEstimator : public ErrorEstimator
         ||∇(u-uₕ)||ₑ ≦ √( C hₑ ∑ₖ (hₖ ∫ |J[∇uₕ]|²) dS )
```

Comment:
Typically, we want both an upper bound and a lower bound.

Reply:
I suggest the following:

```
 It can be roughly described by:
        ||∇(u-uₕ)||ₑ ≅ √( C hₑ ∑ₖ (hₖ ∫ |J[∇uₕ]|²) dS )
    where "e" denotes an element, ||⋅||ₑ the corresponding local norm and k the
    corresponding faces. u is the analytic solution and uₕ the discretized
    solution. hₖ and hₑ are factors dependent on the face and element geometry.
    J is the jump function, i.e. the difference between the limits at each point
    for each side of the face. A custom method to compute hₖ can be provided. It
    is also possible to estimate the error only on a subspace by feeding this
    class an attribute array describing the subspace.

    @note This algorithm is only appropriate for problems with scalar diffusion 
coefficients (e.g., Poisson problems) because it measures only the flux of the
gradients of the discrete solution. The current implementation also does not
include the volume term present in Equation 75 of Kelly et al. Instead, it includes
only the flux term recommended in Equation 82.  The current implementation also 
does not include the constant factor "C".
    It furthermore assumes that the approximation error at the boundary is small
    enough, as the implementation ignores boundary faces.



```

Reply:
Thanks for the suggested documentation. It is now committed.